Symfony/src/AppBundle/Entity/User.php

```diff
@@ -32,6 +38,11 @@ class User implements UserInterface, \Serializable
      * @var string
      *
      * @ORM\Column(name="email", type="string", length=255, unique=true)
+     *
+     * @Assert\Length(
```

Comment:
Instead of asserting the length of the email, why not test if it is a valid email? If I pass in `thisisatest` as input it will be accepted (min length >= 6) but it's not an email.